### PR TITLE
Added check_logged_in and logout session endpoints

### PR DESCRIPTION
**Pull request by thequbit**

### CODE REVIEW

File: tests.py [hunk 1]
```diff
@@ -141,6 +141,7 @@ def run_tests():
         },
         'POST',
     )
+    """
```
Comment (Nolski) [inline tests.py]: Should these be commented out or deleted? If commented out maybe we should put a message as to why they are?

File: yellr-serv/yellrserv/admin_get_views.py [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+        if valid == False:
+            result['error_text'] = "Missing or invalid 'token' field in request."
+            raise Exception('invalid/missing token')
+
+        ret_languages = admin_utils.get_languages()
+
+        result['languages'] = ret_languages
+        result['success'] = True
+
+    except:
```
Comment (Nolski) [inline yellr-serv/yellrserv/admin_get_views.py]: probably shouldn't just throw exceptions into the void here.

File: yellr-serv/yellrserv/admin_get_views.py [hunk 3]
```diff
@@ -0,0 +1,425 @@
...
+        if valid == False:
+            result['error_text'] = "Missing or invalid 'token' field in request."
+            raise Exception('invalid/missing token')
+
+        ret_question_types = admin_utils.get_question_types()
+
+        result['question_types'] = ret_question_types
+        result['success'] = True
+
+    except:
```
Comment (Nolski) [inline yellr-serv/yellrserv/admin_get_views.py]: or here.

**Comment by thequbit** [comment]
Our tests for the Python stuff are in a sad state right now :(... 

I was working on cleaning them and the API docs up,  but got side tracked on kid duty.   I would like to keep them in there for right now until I can clean them up correctly.

**Comment by Nolski** [comment]
It looks like there are sql files being committed. Why is this?

**Comment by thequbit** [comment]
Note: did not mean to close this,  I'm on mobile and fat fingered it. 

The sql file was for the zipcode polygons.   Since we are not going that direction I can remove it.

**Comment by thequbit** [comment]
I can change the thrown exceptions to if statements.  It was a quick way to dump out of the function,  that's all.

**Comment by Nolski** [comment]
I think using exceptions is a good idea :). We should just make sure that they propagate upwards instead of failing silently. Even if that means logging something, or raising an Exception (which you seem to do in other parts).

**Comment by thequbit** [comment]
Agreed.

I think I like the idea of throwing the exception, and then setting the text of the exception to 'error_text' within the json response.

I'll work on a cleaner way of doing things and submit a new PR.

**Comment by thequbit** [comment]
Lots of code clean up in #172  ... covered a lot of the things you mentioned above @Nolski .

**Pull request merged**